consensus/types/src/fork_context.rs

```diff
@@ -82,10 +105,10 @@ impl ForkContext {
         self.digest_to_fork.get(&context)
     }
 
-    /// Returns the context bytes/fork_digest corresponding to a fork name.
-    /// Returns `None` if the `ForkName` has not been initialized.
-    pub fn to_context_bytes(&self, fork_name: ForkName) -> Option<[u8; 4]> {
-        self.fork_to_digest.get(&fork_name).cloned()
+    // TODO: we *may* delete this entire object and just use the spec
```

Comment:
I think it's _possible_ but might be a bit cleaner to keep this for storing current context. Will keep this for now but also open to consider refactoring.